sysl2/sysl/examples/grammars/go/go.gen.sysl

```diff
@@ -26,16 +32,19 @@ CodeGenTransform:
   !view GoStructFields(fields <: set of Type) -> set of FieldDecl:
     fields -> (field:
       let isKeyword = IsKeyword(field.key).out
-      let prefix = if isKeyword == true then "_" else ""
-      identifier = prefix + field.key
+      let fieldName = GetGoField(field.key).out
+      identifier = fieldName
```

Comment:
might be a bug:   do we still need to do `isKeyword` check here on our computed identifier, and rename identifier if it collides with a keyword?  or if collision is no longer possible, why are we calling `IsKeyword`?

Reply:
Hopefully identifiers will not collide with keywords as we capitalize first letter of identifiers. We are not calling keyword anymore